content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,99 @@
+# Persisting Experiments
+
+When you run an experiment with `dvc exp run`, it produces temporary objects in
+the Git repository. These are listed with `dvc exp list` and `dvc exp show`.
+
+Although the experiments as the temporary, they can be shared with
+`dvc exp push` and retrieved from other Git repositories with `dvc exp pull`. In
+this section we'll see how to bring them to the standard Git workflow with
+`dvc exp apply` and `dvc exp branch`.
+
+## Bring Experiment results to your workspace
+
+If it's not run with `--temp` or `--queue` flags, `dvc exp run` leaves the final
+artifacts in your workspace. This is for convenience. However if there are more
+than one experiment you have run, like
+
+```dvc
+$ dvc exp run -S my_param=1
+$ dvc exp run -S my_param=2
+$ dvc exp run -S my_param=3
+```
+
+only the final experiment's results are found in the workspace. In this case,
+you may need to bring the previous experiments' results with `dvc exp apply`.
```

Comment:
```suggestion
Typically, `dvc exp run` leaves the experiment
results in your workspace for convenience. However, you may have run
multiple experiments and wish to go back to a specific one, e.g.

```dvc
$ dvc exp run -S my_param=1
$ dvc exp run -S my_param=2
$ dvc exp run -S my_param=3
```

In this case,
you can restore a previous experiment's results with `dvc exp apply`:
```

Reply:
That's just a copy edit so I'll commit it.